src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/ProviderBase/DbConnectionPool.cs

```diff
@@ -485,7 +485,7 @@ protected override bool ReleaseHandle()
                             DbConnectionPoolIdentity identity,
                             DbConnectionPoolProviderInfo connectionPoolProviderInfo)
         {
-            Debug.Assert(ADP.IsWindowsNT, "Attempting to construct a connection pool on Win9x?");
+            Debug.Assert(ADP.s_isWindowsNT, "Attempting to construct a connection pool on Win9x?");
```

Comment:
I know it's just following the naming rules but it looks bad. Can it be made a property or perhaps just not follow the naming convention in some cases like this for readability?